src/features/utility-bar/store.ts

```diff
@@ -1,22 +1,7 @@
 import { create } from 'zustand'
-import { devtools } from 'zustand/middleware'
+import { devtools, persist, combine } from 'zustand/middleware'
 
-interface UtilityState {
-  isSettingsVisible: boolean
-  setIsSettingsVisible: (newVal: boolean) => void
-
-  isSettingsOpen: boolean
-  setIsSettingsOpen: (open: boolean) => void
-
-  isViewportOpen: boolean
-  setIsViewportOpen: (open: boolean) => void
-
-  isNavBarVisible: boolean
-  setIsNavBarVisible: (visible: boolean) => void
-
-  activeScreenSize: ScreenSize
-  setActiveScreenSize: (newVal: ScreenSize) => void
-}
```

Comment:
Today I learned that Zustand's `combine` helps Typescript infer the types from the store automatically. Now I don't need this duplicative `UtilityState` interface.